smt.go

```diff
@@ -377,6 +377,106 @@ func (smt *SMT) Prove(key []byte) (proof *SparseMerkleProof, err error) {
 	return proof, nil
 }
 
+// ProveClosest generates a SparseMerkleProof of inclusion for the key
+// with the most common bits as the path provided
```

Comment:
I am unsure what you mean here, it cannot bias a child as it follows a preset path, it biases the leaf with the longest common prefix (what we are looking for) but if along the path a `nil` node is hit (a placeholder) then it must have taken a wrong turn (we want a proof of inclusion). For example we took a right child and it was nil, so we go back to the parent and try the left child. Afterwards we continue to follow the path. This can happen either way and I dont see how it could be biased? LMK